nRFMeshProvision/Classes/ProxyFilter.swift

```diff
@@ -410,6 +417,7 @@ internal extension ProxyFilter {
                 guard counter == 0 else {
                     logger?.e(.proxy, "Proxy Filter lost track of devices")
                     counter = 0
+                    callProxyFilterUpdateAcknowledged(type: status.filterType, addresses: addresses)
```

Comment:
My only comment is to this line.
This is called when the proxy responds with a different filter size than the number of addresses sent.
But you still notify about all the addresses, I guess due to no better choice.

Reply:
I replaced it and also removed the call from line 420 so we don't confuse anyone.